### ISSUE TITLE
Callout error: Missing Required prop: "message" when passing message through slot

### ISSUE THREAD

**Issue by nickwade-phl** [CityOfPhiladelphia/phila-ui#137]
The following error / warning is thrown when using the message slot on the Callout component.

![image](https://user-images.githubusercontent.com/46602385/103555940-095f8480-4e7f-11eb-969e-c297e51fac79.png)

Example:

```HTML
<callout type="info">
    <strong>List of Required Documents:</strong>
    <ul>
        <li>Document 1</li>
        <li>Document 2</li>
        <li>Document 3</li>
    </ul>
</callout>
```

**Comment by alejandroxlopez** [comment]
Fix already merged to beta in the initial unit code for phila-ui. PR https://github.com/CityOfPhiladelphia/phila-ui/pull/136/files

**Comment by ibdf** [comment]
I added this fix on it's own commit #139 
Fixed in version 2.0.1
